schema.sql

```diff
@@ -58,3 +57,43 @@ ALTER TABLE animals
 ADD CONSTRAINT fk_owner
 FOREIGN KEY (owner_id)
 REFERENCES owners(id);
+
+
+
+CREATE TABLE vets(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR NOT NULL,
+    age INT,
+    date_of_graduation DATE,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE specializations(
+    vet_id INT NOT NULL,
+    species_id INT NOT NULL
+);
+
+ALTER TABLE specializations
+ADD CONSTRAINT fk_vet
+FOREIGN KEY (vet_id)
+REFERENCES vets(id);
+
+ALTER TABLE specializations
+ADD CONSTRAINT fk_species
+FOREIGN KEY (species_id)
+REFERENCES species(id);
+
+CREATE TABLE visits(
+    vet_id INT NOT NULL,
+    animal_id INT NOT NULL
+)
+
```

Comment:
- [ ] Great work on making this join table but kindly double check the requirements because it should have one more column with the name of visit_date or any name you wanna choose for the visits of the vets with animals 👍 to follow the project requirements 😊